.gitignore

```diff
@@ -45,7 +45,7 @@ build
 /build*/
 /Build*/
 /debug*/
-/release*/
+/release?*/
```

Comment:
FWIW the new files weren't added because of this pattern in the `.gitignore` file.  It was matching any directories that started with `/release`, including `/release/` itself.  I had hoped it would only match folders that started with release but had added strings afterward.  Anyway, I modified the pattern so that it would match `release` plus at least one character.  Immediately after modifying this pattern, I was able to add the `in.html` files without having to force anything.